### PR TITLE
Positioning and Floating Elements

### PR DESCRIPTION
**Pull request by mricanho**
In this project we cloned an article of the New York Times (https://nyti.ms/3e39LZP), the primary goal was to get all the elements aligned with a combination of all three techniques (float, flex, and grid), plus the correct use of HTML semantics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+<h2>Authors</h2>
+<p><strong>Gustavo Sanmartin</strong></p>
+<ul>
+  <li>Github: <a href="https://github.com/gasb150">@JTavoSanmartin</a>
+</ul>
+<p><strong>Miguel Ricaño</strong></p>
+<ul>
+  <li>Github: <a href="https://github.com/mricanho">@MiguelRicaño</a>
+</ul>
```
Comment (arslanbisharat) [inline README.md]: - [x] [Optional] You can add more social links here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,856 @@
...
+<body>
+  <header>
+    <div class="container">
+      <nav class="nav-bar">
+        <div class="left-side">
+          <span class="iconify menu pointer" data-icon="codicon:three-bars" data-inline="false">
+          </span>
+          <span class="iconify loupe pointer" data-icon="bx:bx-search" data-inline="false">
+          </span>
+          <p>SPACE & COSMOS</p>
```
Comment (arslanbisharat) [inline index.html]: - [x] Change this text to link

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+
+.paid-st {
+  font-size: 8px;
+  font-weight: bolder;
+}
+
+#paid-logo {
+  height: 25px;
+  width: auto;
+}
```
Comment (arslanbisharat) [inline css/style.css]: - [x] Please don't use ids as selectors

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,619 @@
...
+
+footer {
+  display: block;
+  align-items: center;
+  margin-left: 18%;
+  margin-right: 18%;
+  border-top: 1px solid rgb(243, 243, 243);
+  margin-top: 15px;
+  margin-bottom: 20px;
```
Comment (arslanbisharat) [inline css/style.css]: - [x] Excluding border-top, the properties margin-top, margin-bottom, margin-left, margin-right can be replaced by the margin.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳

_Hello Team,_

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

**Please have a look at the suggestions to get more understanding.**

- [x] You should change **SPACE & COSMOS** to a link. 

**Clone**

chrome-capture (38)

- [x] Footer should have 100% width. But your clone has centered items. Please have a look here. 

**Clone**

chrome-capture (40)

**Original**

chrome-capture (41)

- [x] Footer should be sticky. But in your clone, it is not as present on the original page. 

- [x] Please use dry CSS. there are too many font-size declarations (34), abstraction needed

- [x] [Optional] You can add a brief description of 2-3 lines in the about section offy our project. 


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: README.md [hunk 5]
```diff
@@ -1 +1,40 @@
...
+<p>Contributions, issues, and feature requests are welcome!<p>
+<p>Feel free to check the <a href="https://github.com/mricanho/p-f-elements/issues/1">issues page.</a></p>
+ 
+<h2>Show your support</h2>
+<p> Give a 
+  <g-emoji class="g-emoji" alias="star" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/2b50.png"><img class="emoji" alt="star" height="20" width="20" src="https://github.githubassets.com/images/icons/emoji/unicode/2b50.png"></g-emoji>
+  if you like this project!</p>
+  
+<h2>License</h2>
+  <p>This project is <a href="../featureDev/LICENSE">MIT</a> licensed.</p>
```
Comment (descholar-ceo) [inline README.md:40]: Try to use markdown language on all `.md` files, don't use HTML please,

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @mricanho  and @gasb150 , your project is approved :tada: :tada: :tada: 
You have implemented all of the requested changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**